apps/client/src/pages/ConclusionPage.js

```diff
@@ -99,6 +99,68 @@ const ConclusionPage = ({ topic, checker }) => {
   };
   const currentDateTime = date.toLocaleDateString("nl-NL", options);
 
+  // Demo saveConclusion function
+  const saveConclusion = () => {
```

Comment:
I think we are saving more then conclusions, so maybe better to name it something like `savePermitResults`, `saveCheckerResult` ?

Reply:
When do we call this function? Every time we show the conclusion page?

Reply:
From where would we call this function? Did you test with mutations?

Reply:
Good questions.

This function is called:
- every time the conclusion page is loaded (without user data)
- every time the user wants to save the progress (including user data)
- every time the user wants to save the conclusion (including user data)

This function is called from another file. This is just a sketch.

I did not test with mutations yet